### PR TITLE
be more specific in the zcml condition

### LINKED ISSUE #144: plone.app.multilingual installation detection code fails in Plone 4 and p.a.m 12.x
The plone.app.multilingual detection code in `import_others` checks that [ITranslationManager](https://github.com/collective/collective.exportimport/blob/main/src/collective/exportimport/import_other.py#L50) interface exists, which is not true for p.a.multilingual 1.2.x.

So when trying to use collective.exportimport in Plone 4 with p.a.multilingual 1.2.x, starting the instance fails because it tries to register the `import_translations` view https://github.com/collective/collective.exportimport/blob/main/src/collective/exportimport/configure.zcml#L163-L169

The point is that the `zcm:condition="installed plone.app.multilingual"` is true but there is no `ImportTranslations` view in the `import_others` file because `HAS_PAM` is false there.

### PR DESCRIPTION
**Pull request by erral**
Fixes #144 

This way the endpoint is registered only when `ITranslationManager` exists in p.a.multilingual, what happens in versions >= 2.

This way this addon can be used in Plone 4.3.x sites with p.a.multilingual 1.2.x. In such cases we just want to export the content, so we should not bother about not being able to import the translations.

### CODE REVIEW

File: src/collective/exportimport/configure.zcml [hunk 1]
```diff
@@ -161,7 +161,7 @@
       />
 
   <browser:page
-      zcml:condition="installed plone.app.multilingual"
+      zcml:condition="installed plone.app.multilingual.interfaces.ITranslationManager"
```
Comment (davisagli) [inline src/collective/exportimport/configure.zcml:164]: @erral Unfortunately I think this only works with a module or package and won't work with an object inside a module namespace. The `installed` check is implemented here using `__import__`: https://github.com/zopefoundation/zope.configuration/blob/master/src/zope/configuration/xmlconfig.py#L367

and this is what happened when I tried to import ITranslationManager that way:
```
>>> __import__('plone.app.multilingual.interfaces.ITranslationManager')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'plone.app.multilingual.interfaces.ITranslationManager'; 'plone.app.multilingual.interfaces' is not a package
```

**Review by davisagli — CHANGES_REQUESTED** [review]

File: src/collective/exportimport/configure.zcml (hunk 1 above)
Comment (davisagli) [reply src/collective/exportimport/configure.zcml:164]: Maybe you can add a conditional inside the view code instead?

Comment (erral) [reply src/collective/exportimport/configure.zcml:164]: And how did it worked for me? :/

**Comment by erral** [comment]
I reworked this setting a dummy class for the `import_translations` view returning an informative message

File: src/collective/exportimport/configure.zcml (hunk 1 above)
Comment (erral) [reply src/collective/exportimport/configure.zcml]: It worked because I didn't have it enabled in my buildout :roll_eyes:

**Review by pbauer — APPROVED** [review]

**Pull request merged**